src/index.html

```diff
@@ -4,9 +4,354 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a href="#home">
+        <img 
+          src="./images/logo.png" 
+          class="logo" 
+          alt="Logo MOYO"
+        >
+      </a>
+      
+      <nav class="nav">
+        <ul class="nav__list">
+          <li>
+            <a href="#apple" class="nav__link is-active">Apple</a>
+          </li>
+
+          <li>
+            <a href="#samsung" class="nav__link">Samsung</a>
+          </li>
+
+          <li>
+            <a href="#smartphones" class="nav__link">Smartphones</a>
+          </li>
+
+          <li>
+            <a 
+              href="#laptops&computers" 
+              data-qa="nav-hover" 
+              class="nav__link"
+            >
+              Laptops & computers
+            </a>
+          </li>
+
+          <li>
+            <a href="#gadgets" class="nav__link">Gadgets</a>
+          </li>
+
+          <li>
+            <a href="#tablets" class="nav__link">Tablets</a>
+          </li>
+          
+          <li>
+            <a href="#photo" class="nav__link">Photo</a>
+          </li>
+          <li>
+            <a href="#video" class="nav__link">Video</a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+    <main class="cards-catalog">
+      <div class="cards-container">
+        <div class="card" data-qa="card">
+          <img
+            src="./images/imac.jpeg"
+            class="card__image"
+            alt="Image Apple gadget"
+          >
+          <h2 class="card__description-link">
+            APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+          </h2>
+        
+          <p class="card__product-code-link">
+            Product code: 195434
+          </p>
+          <div class="card__stars stars">
+            <div class="stars__container stars__container--4">
```

Comment:
```suggestion
          <div class="card__stars stars stars--4">
```
Do you really need a container? 
Just apply all the outer styles to card__stars